### PR TITLE
Improve wording: Purge -> Clear

### PR DESCRIPTION
**Pull request by smehrbrodt**
This string is only in the `3.5-dev` branch and thus it was not changed with 6f9a1314395288aca6a0a48a12dac96369d076ef

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_languages.ini [hunk 1]
```diff
@@ -109,8 +109,8 @@ COM_LANGUAGES_VIEW_OVERRIDES_LANGUAGES_BOX_ITEM="%1$s - %2$s"
 COM_LANGUAGES_VIEW_OVERRIDES_N_ITEMS_DELETED="%d language overrides were successfully deleted."
 COM_LANGUAGES_VIEW_OVERRIDES_N_ITEMS_DELETED_1="%d language override was successfully deleted."
 COM_LANGUAGES_VIEW_OVERRIDES_NO_ITEM_SELECTED="You haven't selected any overrides."
-COM_LANGUAGES_VIEW_OVERRIDES_PURGE="Purge"
-COM_LANGUAGES_VIEW_OVERRIDES_PURGE_SUCCESS="Overrider cache table successfully purged."
+COM_LANGUAGES_VIEW_OVERRIDES_PURGE="Clear Cache"
+COM_LANGUAGES_VIEW_OVERRIDES_PURGE_SUCCESS="Overrides cache table successfully cleared."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_languages.ini]: There was discussion about the use of the word Overrider when this was committed originally and as the name of the table is #__overrider (perhaps a mistake orginaly) then it was decided that the string should be the name of the table

Comment (smehrbrodt) [inline administrator/language/en-GB/en-GB.com_languages.ini]: Then what about these lines?

` COM_LANGUAGES_VIEW_OVERRIDES_NO_ITEM_SELECTED="You haven't selected any overrides."`
` COM_LANGUAGES_VIEW_OVERRIDES_TITLE="Language Manager: Language Overrides"`

At least it should be consistent.

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_languages.ini]: They are overrides it is just the name of the table that is wrong so when you say the xxx table has been cleared it should be the name of the table

Not sure I agree 100% but that was the decision

Comment (smehrbrodt) [inline administrator/language/en-GB/en-GB.com_languages.ini]: Should we then fix the table name? I guess it must have been a mistake to name it like that.

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_languages.ini]: thats a much bigger task especially for existing sites

Comment (smehrbrodt) [inline administrator/language/en-GB/en-GB.com_languages.ini]: Ok, then I'll revert this one back to "Overrider" for now.
Still think the table should be renamed. Should be doable with an upgrade script, like any other db changes are done.

**Comment by brianteeman** [comment]
Looks good to me

File: administrator/language/en-GB/en-GB.com_languages.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_languages.ini]: feel free to create a new issue about the table name

On 9 July 2015 at 21:25, Samuel Mehrbrodt <notifications@github.com> wrote:

> In administrator/language/en-GB/en-GB.com_languages.ini
> <https://github.com/joomla/joomla-cms/pull/7380#discussion_r34299565>:
>
> > @@ -109,8 +109,8 @@ COM_LANGUAGES_VIEW_OVERRIDES_LANGUAGES_BOX_ITEM="%1$s - %2$s"
> >  COM_LANGUAGES_VIEW_OVERRIDES_N_ITEMS_DELETED="%d language overrides were successfully deleted."
> >  COM_LANGUAGES_VIEW_OVERRIDES_N_ITEMS_DELETED_1="%d language override was successfully deleted."
> >  COM_LANGUAGES_VIEW_OVERRIDES_NO_ITEM_SELECTED="You haven't selected any overrides."
> > -COM_LANGUAGES_VIEW_OVERRIDES_PURGE="Purge"
> > -COM_LANGUAGES_VIEW_OVERRIDES_PURGE_SUCCESS="Overrider cache table successfully purged."
> > +COM_LANGUAGES_VIEW_OVERRIDES_PURGE="Clear Cache"
> > +COM_LANGUAGES_VIEW_OVERRIDES_PURGE_SUCCESS="Overrides cache table successfully cleared."
>
> Ok, then I'll revert this one back to "Overrider" for now.
> Still think the table should be renamed. Should be doable with an upgrade
> script, like any other db changes are done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/7380/files#r34299565>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by Kubik-Rubik** [comment]
Thank you @smehrbrodt! Merged into 3.5-dev branch.

**Pull request closed without merging**